packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -24,45 +18,33 @@ const SubscriptionUtils = (module.exports = {
     ...plan.plan_metadata,
   }),
 
+  // Parse a comma-separated list of capabilities with allowance for varied whitespace
+  splitCapabilities: s =>
+    (s || '')
+      .trim()
+      .split(',')
+      .map(c => c.trim())
+      .filter(c => !!c),
+
   determineClientVisibleSubscriptionCapabilities: async function(
-    config,
```

Comment:
Turns out that we don't need access to config, database, or request auth to determine capabilities directly from Stripe metadata